test/unit/Support.cpp

```diff
@@ -224,7 +219,7 @@ uint32_t Support::diff_files(std::istream& str1, std::istream& str2,
     uint32_t numdiffs = 0;
     char p, q;
 
-    for (uint32_t i = 0; true; ++i)
+    for (uint32_t i = 0; ; ++i)
```

Comment:
This is certainly one of the smaller ones. We could gloss over it, suppress it, whatever. The overall intent of the PR is to get the Windows build output to be significantly quieter so that we can see the actual errors through the slew of warnings.